src/main/java/com/qualfacul/hades/conversation/ConversationDTO.java

```diff
@@ -0,0 +1,75 @@
+package com.qualfacul.hades.conversation;
+
+import java.util.Calendar;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.qualfacul.hades.serialization.FullDateCalendarSerializer;
+
+public class ConversationDTO {
+	
+	private Long id;
+	
+	private Long collegeId;
+	
+	@JsonSerialize(using = FullDateCalendarSerializer.class)
+	private Calendar createdAt;
```

Comment:
que tal usar um `LocalDateTime`?
se for usar, precisa criar um converter, vide `LocalDateAttributeConverter`
e vai ter que trocar o Serializer